models/cm_compartmentcurrents.cpp

```diff
@@ -50,6 +50,40 @@ nest::Na::Na( const DictionaryDatum& channel_params )
   }
 }
 
+void
+nest::Na::pre_run_hook( const double v_init )
+{
+  // compute limiting values for m and h given v_init as the initial state
```

Comment:
This looks a bit like it should rather be a Doxygen-style comment for the function in the header file. Also the one below with the link to the reference.

Reply:
I have implemented this comment style (while also following @med-ayssar 's comment to refactor the state variable activations into their own functions)